SPECS/memtest86+.spec

```diff
@@ -36,5 +36,8 @@ install -m644 memtest.{bin,efi} -D %{buildroot}/boot/
 /boot/memtest.bin
 
 %changelog
+* Fri Jan 6 2023 Yann Dirson <yann.dirson@vates.fr> - 6.01-2
```

Comment:
The date should be that of the packaging change, so today, and there is a convention (though I don't know where it's documented) to pad the month number with a 0 (applies to the previous entry too).